internal/pkg/build/build.go

```diff
@@ -298,7 +298,11 @@ func (b *Build) Full() error {
 			a.HandleSection(k, v)
 		}
 
-		a.HandleBundle(stage.b)
+		err := a.HandleBundle(stage.b)
+		if err != nil {
+			return err
```

Comment:
Lets wrap this with something a like "while inserting app metadata:..." or something